cfme/common/vm.py

```diff
@@ -374,7 +374,10 @@ def open_console(self, console='VM Console', invokes_alert=False, cancel=False):
         view = navigate_to(self, 'Details')
 
         # Click console button given by type
-        view.toolbar.access.item_select(console)
+        if invokes_alert:
```

Comment:
**REQUIRED**: Given this is ```False``` by default and I'm thinking ```True``` if not, why can't we just do

```view.toolbar.access.item_select(console, handle_alert=invokes+alert)```